src/http/api/derepost.ts

```diff
@@ -111,7 +110,6 @@ export function createDerepostActionHandler(
         await ctx.get('globaldb').set([undo.id!.href], undoJson);
 
         // Remove announce activity from database
-        await removeFromList(ctx.get('db'), ['reposted'], announceId.href);
```

Comment:
Note for reviewer: this `'reposted'` collection doesn't seem to be in use:
- we never write to it
- we read from it in the Inbox collection (see `inboxHandler` function), but that code is now also deleted

So don't think we have any reason to keep those `removeFromList` calls for the `reposted` collection